docs/dev/tutorials/aa-daily-spend-limit.md

```diff
@@ -915,7 +1029,7 @@ To make a transfer, run the following:
 yarn hardhat deploy-zksync --script deploy/transferETH.ts
 ```
 
-Although the error message doesn't give us any concrete reason, it's anticipated that the transaction was reverted like the below:
+Although the error message doesn't give us the specifics, we know it failed because the amount exceeded the limit. You should see following output:
```

Comment:
```suggestion
You should see an error message. Although the error message doesn't give us the specifics, we know it failed because the amount exceeded the limit.
```